dist-git.spec

```diff
@@ -165,6 +165,7 @@ ln -s %{_datadir}/dist-git/setup_git_package %{buildroot}%{_bindir}/setup_git_pa
 ln -s %{_datadir}/dist-git/mkbranch %{buildroot}%{_bindir}/mkbranch
 ln -s %{_datadir}/dist-git/mkbranch_branching %{buildroot}%{_bindir}/mkbranch_branching
 ln -s %{_datadir}/dist-git/remove_unused_sources %{buildroot}%{_bindir}/remove_unused_sources
+ln -s %{_datadir}/dist-git/garbage_collection %{buildroot}%{_bindir}/garbage_collection
```

Comment:
The whole `ln -s` is ugly hack. Probably because the history. Since this is a new thing can we simply move it from datadir to bindir?